scripts/resume.js

```diff
@@ -0,0 +1,44 @@
+const resume = {
```

Comment:
Love the comments, especially since I am new to programming and reading other's code 
This helps so much to solidify